lms/templates/staff_problem_info.html

```diff
@@ -51,9 +51,9 @@
     </form>
 
   </div>
-</section>
+</div>
 
-<section aria-hidden="true" class="modal staff-modal" id="${element_id}_debug" style="width:80%; left:20%; height:80%; overflow:auto; display:none" >
+<div aria-hidden="true" class="modal staff-modal" id="${element_id}_debug" style="width:80%; left:20%; height:80%; overflow:auto; display:none" >
```

Comment:
As part of TNL-5671 i just changed following bullet, rest are strikethrough so i've not considered them as part of acceptance criteria.

> Move - aria-labelledby on <div class="problem" TO <div class="problems-wrapper" AND add a role="group to <div class="problems-wrapper"